src/Plugin/Field/FieldFormatter/AvPortalVideoFormatter.php

```diff
@@ -234,6 +246,7 @@ public function viewElements(FieldItemListInterface $items, $langcode): array {
   protected function viewElement(FieldItemInterface $item): array {
     $max_width = $this->getSetting('max_width');
     $max_height = $this->getSetting('max_height');
+    $autoplay_setting = $this->getSetting('autoplay_setting') == 0 ? FALSE : TRUE;
```

Comment:
The setting is a boolean, why we compare it with 0?

Reply:
I see below that autoplay was passed as string `true`, maybe we should convert to strings true/false?